dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -1094,19 +1051,42 @@ private void preprocessListValues(List<Object> cwl, String currentPath, String v
             cwl.replaceAll(v -> preprocess(v, currentPath, version, depth));
         }
 
-        private boolean isEntry(Map<String, Object> cwl) {
-            String c = (String)cwl.get("class");
-            return Objects.equals(c, "CommandLineTool") || Objects.equals(c, "ExpressionTool") || Objects.equals(c, "Workflow");
+        private boolean isProcess(Map<String, Object> cwl) {
+            Object c = cwl.get("class");
+            return "Workflow".equals(c) || "CommandLineTool".equals(c) || "ExpressionTool".equals(c) || "Operation".equals(c);
```

Comment:
It was like this, but thinking of edge cases, is the class case-sensitive? In other words, could it return "workflow"?

Reply:
I think CWL is generally case sensitive, and since we're trying to encourage interoperability, I'd generally lean towards a strict parse up front.  However, that's moot, because the cwljava parser, which is invoked directly after the preprocessor, will fail to parse if `Workflow` is not capitalized correctly...